### ISSUE TITLE
Idlemove.py is not moving the user back to the original channel

### ISSUE THREAD

**Issue by r-vd-woude** [mumble-voip/mumo#13]
As stated above, the user does not move back to the default channel. The log says this:

```
2018-03-26 17:40:47,255 INFO idlemove 13s > 10s: State transition for user Moonlighter (76/1) from mute False -> True / deaf False -> False | channel 7 -> 12 on server 1
2018-03-26 17:40:50,990 INFO idlemove Restore user Moonlighter (76/1) on server 1
2018-03-26 17:41:07,272 INFO idlemove 11s > 10s: State transition for user Moonlighter (76/1) from mute False -> True / deaf False -> False | channel 12 -> 12 on server 1
2018-03-26 17:41:13,716 INFO idlemove Restore user Moonlighter (76/1) on server 1
2018-03-26 17:41:27,282 INFO idlemove 11s > 10s: State transition for user Moonlighter (76/1) from mute False -> True / deaf False -> False | channel 12 -> 12 on server 1
2018-03-26 17:47:30,571 INFO idlemove Restore user Moonlighter (76/1) on server 1
2018-03-26 17:47:47,363 INFO idlemove 14s > 10s: State transition for user Moonlighter (76/1) from mute False -> True / deaf False -> False | channel 12 -> 12 on server 1
2018-03-26 17:48:15,288 INFO idlemove Restore user Moonlighter (76/1) on server 1
2018-03-26 17:48:27,374 INFO idlemove 12s > 10s: State transition for user Moonlighter (76/1) from mute False -> True / deaf False -> False | channel 2 -> 12 on server 1
2018-03-26 17:48:30,518 INFO idlemove Restore user Moonlighter (76/1) on server 1
```

(I set the timer to 10 seconds to reproduce this problem.)

However, when it says restore user Moonlighter, it still does not place the user back to the original channel.

**Comment by raku-cat** [comment]
Hey, can you try my pull request and see if it fixes this for you?

**Comment by Kissaki** [comment]
The user is now restored to the previous channel.

But that means if the activity is a channel join, then the user gets moved to the previous channel right after joining the channel they chose to. I think that's unintuitive behavior.

/edit: Fixed in f99a20f41350c212eae123d4898ceb4df93c06e2
